client-app/src/mobile/grids/GridPageModel.js

```diff
@@ -19,23 +19,38 @@ export class GridPageModel {
     loadModel = new PendingTaskModel();
 
     gridModel = new GridModel({
+        stateModel: 'toolboxSampleGrid',
+        sortBy: ['profit_loss|desc|abs'],
         store: new LocalStore({
-            fields: ['company', 'profit_loss']
+            fields: ['company', 'city', 'trade_volume', 'profit_loss']
         }),
-        leftColumn: {
-            headerName: 'Company',
-            field: 'company'
-        },
-        rightColumn: {
-            headerName: 'P&L',
-            field: 'profit_loss',
-            valueFormatter: numberRenderer({
-                precision: 0,
-                ledger: true,
-                colorSpec: true,
-                asElement: true
-            })
-        }
+        subFields: [
```

Comment:
where do I specify if I actually want the label for the SubField or not?  That was an important feature in zoned grid.  Also, whatever label we do have, can probably default to the field name, in StartCase()